packages/node-core/src/indexer/testing.service.ts

```diff
@@ -216,10 +231,54 @@ export abstract class TestingService<A, SA, B, DS> {
       this.totalFailedTests += test.expectedEntities.length;
       logger.warn(e, `Test ${test.name} failed to run`);
     } finally {
+      await this.dropSchema(schema);
+    }
+  }
+
+  private async dropSchema(schema: string) {
+    const modelsRelation = getAllEntitiesRelations(this.project.schema);
+
+    try {
+      // drop existing project schema and metadata table
       await this.sequelize.dropSchema(`"${schema}"`, {
         logging: false,
         benchmark: false,
       });
+
+      // TODO, remove this soon, once original enum are cleaned
+      // Deprecate, now enums are moved under schema, drop schema will remove project enums
+      await Promise.all(
+        modelsRelation.enums.map(async (e) => {
+          const enumTypeNameDeprecated = `${schema}_enum_${enumNameToHash(e.name)}`;
+          const resultsDeprecated = await getEnumDeprecated(this.sequelize, enumTypeNameDeprecated);
+          if (resultsDeprecated.length !== 0) {
+            await this.sequelize.query(`
+            DROP TYPE "${enumTypeNameDeprecated}";
+          `);
+          }
+        })
+      );
+
+      // remove schema from subquery table (might not exist)
+      const checker = await this.sequelize.query(
```

Comment:
Again, this shouldn't exist either. What's causing this to exist?

Reply:
I copied the logic from force-clean